manim/mobject/three_dimensions.py

```diff
@@ -47,10 +47,8 @@ def construct(self):
                 axes = ThreeDAxes(x_range=[-4,4], x_length=8)
                 surface = ParametricSurface(
                     lambda u, v: axes.c2p(*self.func(u, v)),
-                    u_min=-PI,
-                    u_max=PI,
-                    v_min=0,
-                    v_max=TAU,
+                    u_range=[-PI, PI],
+                    v_range=[0, TAU]
```

Comment:
If possible, I we should try to add some docs for at least these two parameters.